### PR TITLE
Replace UserProfile

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: src/enumerate.rs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        };
+        Ok(&VALUE)
+    }
+
+    #[classmethod]
+    fn iter(_cls: &PyType) -> PyResult<[UserProfile; 3]> {
+        Ok([
+            Self(libparsec::types::UserProfile::Admin),
+            Self(libparsec::types::UserProfile::Standard),
+            Self(libparsec::types::UserProfile::Outsider),
```
Comment (FirelightFlagboy) [inline src/enumerate.rs:123]: Why not using method `admin()`, `standard()` & `outsider()` here ?

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: src/enumerate.rs (hunk 1 above)
Comment (FirelightFlagboy) [reply src/enumerate.rs]: @TimeEngineer up

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]
Just a previous discussion that wasn't resolved

File: src/enumerate.rs (hunk 1 above)
Comment (TimeEngineer) [reply src/enumerate.rs]: @FirelightFlagboy Outdated

**Review by FirelightFlagboy — APPROVED** [review]

File: tests/schemas/builder.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def field_to_spec(field):
         # This is done to easily see which schema is impacted on enum
         # modification (for instance adding a value in an enum doesn't impact
         # a schema using an `EnumCheckedConstant` on this enum).
-        spec["enum_allowed_values"] = [v.value for v in field.ENUM]
+        spec["enum_allowed_values"] = [v.value for v in field.ENUM]  # .values()]
```
Comment (FirelightFlagboy) [inline tests/schemas/builder.py:73]: Why this comment ?

Comment (TimeEngineer) [reply tests/schemas/builder.py:73]: Good catch

**Review by FirelightFlagboy — APPROVED** [review]

**Review by FirelightFlagboy — APPROVED** [review]

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**